octobot/community/feeds/community_ws_feed.py

```diff
@@ -171,7 +171,7 @@ async def _fetch_stream_identifier(self, identifier):
         if identifier is None:
             return None
         async with self.authenticator.get_aiohttp_session().get(
-                f"{constants.OCTOBOT_COMMUNITY_FETCH_FEED_IDENTIFIER_URL}/{identifier}") as resp:
+                f"{constants.OCTOBOT_COMMUNITY_URL}api/v2/storefront/feeds/id/{identifier}") as resp:
```

Comment:
That's a temporary url right?

Reply:
It's a legacy one, we don't use `community_ws_feed` anymore

Reply:
Right